### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lucyzhang929**
Tracking issue here: https://github.com/Azure/azure-functions-sql-extension/issues/654

### CODE REVIEW

**Review by VasuBhog — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Databases on SQL Server, Azure SQL Database, or Azure SQL Managed Instance which
 ## Known Issues
 - The table used by a SQL binding or SQL trigger cannot contain two columns that only differ by casing (Ex. 'Name' and 'name'). 
 - Non-CSharp functions using SQL bindings against tables with columns of data types `BINARY` or `VARBINARY` need to map those columns to a string type. Input bindings will return the binary value as a base64 encoded string. Output bindings require the value upserted to binary columns to be a base64 encoded string.
+- SQL Bindings against tables with columns of data types `GEOMETRY` and `GEOGRAPHY` are not supported.
```
Comment (VasuBhog) [inline README.md:45]: Would it be good to include the issue link here for users to go to?

**Review by Charles-Gagnon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Charles-Gagnon) [reply README.md:45]: Would be good thing to do for all these that we have tracking issues for actually. @lucyzhang929 do you have a bit of spare time to do that?

Comment (chlafreniere) [reply README.md:45]: Agreed if we can put the issue, that'd be great.

Also, please lowercase "Bindings" -> "bindings" (see other examples above).

Comment (lucyzhang929) [reply README.md:45]: Yep I can link the issues

File: README.md [hunk 2]
```diff
@@ -58,9 +58,6 @@ Databases on SQL Server, Azure SQL Database, or Azure SQL Managed Instance which
   - PowerShell: The workaround is to use the `$TriggerMetadata[$keyName]` to retrieve the query property - an example can be found [here](https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-powershell/AddProductParams/run.ps1). Issue is tracked [here](https://github.com/Azure/azure-functions-powershell-worker/issues/895).
   - Python: The workaround is to use `parse_qs` - an example can be found [here](https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-python/AddProductParams/__init__.py). Issue is tracked [here](https://github.com/Azure/azure-functions-python-worker/issues/894).
 
-### Input Bindings
-- For Java Functions using Output bindings against a table with columns of data types 'DATETIME', 'DATETIME2', or 'SMALLDATETIME', use java.util.Date type to ensure the datetime is formatted correctly.
```
Comment (lucyzhang929) [inline README.md:62]: Meant to remove this when adding known issues to Java guide https://github.com/Azure/azure-functions-sql-extension/blob/main/docs/SetupGuide_Java.md#known-issues

**Pull request merged**